src/buildQueryURL.ts

```diff
@@ -192,6 +203,20 @@ export interface QueryParams {
 	 * {@link https://prismic.io/docs/route-resolver}
 	 */
 	brokenRoute?: string
+
+	/**
+	 * Whether or not the client is running in a development environment.
+	 *
+	 * @internal
+	 */
+	[PRISMIC_DEV_PARAM]?: number
+
+	/**
+	 * The client version used to make the request.
+	 *
+	 * @internal
+	 */
+	[PRISMIC_CLIENT_VERSION_PARAM]?: string
```

Comment:
💡 #idea: Rather than accept these as arguments, it might make more sense for `buildQueryURL()` to inject these automatically. WDYT?

Reply:
Agreed, refactored!